python/whl.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash -e
+
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# TODO(mattmoor): Ideally we'd rewrite this in Python for portability, but
+# since the executes during WORKSPACE instantiation that would require
+# redistributing PAR files, since we cannot invoke built tools at that time.
+WHL="$1"
+REQUIREMENTS="$2"
+PKG=$(basename "${WHL}" | cut -d'-' -f 1)
+DIST_INFO="$(basename "${WHL}" | cut -d'-' -f -2).dist-info"
```

Comment:
`# E.g. google_cloud-0.27.0-py2.py3-none-any.whl` -> `google_cloud-0.27.0.dist-info`

Reply:
Added to `whl.py`